examples/basic-app/model/common-ent.jsonic

```diff
@@ -0,0 +1,34 @@
+sys: &: $.main.shape.ent
+
+sys: task: {
+  fields: {
```

Comment:
I broke my own rule here - this should be  `field` singular

Reply:
the general shape of a field should be defined at a higher level in $.main.shape.ent
try:
```
main: shape: ent: field: &: {
  label: *'ID' | string
  required: *true | boolean
  ...
}
```

Reply:
Done